### DISCUSSION TITLE
Applying app layout only to open applications

### DISCUSSION THREAD

**Discussion by naftalibeder** [rxhanson/RectanglePro-Community#380] [Q&A] [answered]
Is there any way to apply a layout to exclusively open applications, without opening ones in the layout configuration that are closed? Right now, if I have apps A and B in the configuration, but only A is open, and I apply the layout, B gets opened.

**Comment by rxhanson** [comment, marked as answer]
Sounds like you might have the "Launch closed/minimized apps" checkbox checked for your layout? If you click on row for the layout, you can adjust this.

**Reply by naftalibeder** [reply]
Interesting - it wasn't checked, but I checked and then unchecked it, and now it's working as expected. I'm not able to reproduce the issue now, so I think we can consider this closed. Thanks!
